lookup_tables/greynoise/advanced/noise_advanced.yml

```
@@ -117,6 +117,9 @@ LogTypeMap:
         - "$.protoPayload.requestMetadata.callerIP"
         - "$.httpRequest.remoteIP"
         - "$.httpRequest.serverIP"
+    - LogType: Github.Audit
```

```suggestion
    - LogType: GitHub.Audit
```

Other references to this logtype are spelled `GitHub.Audit`